addons/website_mail_channel/static/src/snippets/s_channel/options.js

```diff
@@ -8,12 +8,38 @@ var wUtils = require('website.utils');
 var _t = core._t;
 
 options.registry.Channel = options.Class.extend({
+    publicChannels: [],
```

Comment:
You can remove that. If you wanted to initialize it, you should do it in the `init` method but this is useless anyway